test/model/binding.dart

```diff
@@ -513,34 +512,6 @@ class FakeFlutterLocalNotificationsPlugin extends Fake implements FlutterLocalNo
     return true;
   }
 
-  FlutterLocalNotificationsPlatform? _platform;
-
-  @override
-  T? resolvePlatformSpecificImplementation<T extends FlutterLocalNotificationsPlatform>() {
```

Comment:
commit-structure nit: this should come as a separate commit after the main commit, as per https://github.com/zulip/zulip-flutter/pull/795#discussion_r1684959410